skypy/halos/_colossus.py

```diff
@@ -203,3 +203,137 @@ def colossus_mf(redshift, model, mdef, m_min, m_max, sky_area, cosmology,
                                         cosmology, sigma8, ns, size, resolution)
 
     return z, m
+
+
+def concentration(mass, mdef, redshift, model, cosmology, sigma8, ns):
+    r'''Halo concentration calculator.
+
+    This function calculates halo concentration(s) using the model of c-M relation
+    available in colossus.
+
+    Parameters
+    ----------
+    mass : float or array_like
+        Spherical overdensity halo mass in units of solar mass/h, corresponding
+        to the mass definition, mdef.
+    mdef : str
+        Halo mass definition for spherical overdensities used by colossus.
+    redshift : float
+        Halo redshift
+    model : string
+        The model of the c-M relation which is available in colossus.
+    cosmology : astropy.cosmology.Cosmology
+        Astropy cosmology object
+    sigma8 : float
+        Cosmology parameter, amplitude of the (linear) power spectrum on the
+        scale of 8 Mpc/h.
+    ns : float
+        Cosmology parameter, spectral index of scalar perturbation power spectrum.
+
+    Returns
+    -------
+    concentration : float or array_like
+        Halo concentration(s); has the same dimensions as mass.
+
+    '''
+    from colossus.cosmology.cosmology import fromAstropy
+    from colossus.halo import concentration
+
+    fromAstropy(cosmology, sigma8=sigma8, ns=ns)
+
+    c = concentration.concentration(mass, mdef, redshift, model)
+
+    return c
+
+
+def radius(mass, concentration, redshift, mdef, Delta, cosmology, sigma8, ns):
```

Comment:
Is this function name descriptive enough? what about `radii_NFW` or anything better??